css/index.css

```diff
@@ -1 +1,487 @@
-/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
\ No newline at end of file
+/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+     font-size: 62.5%; 
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+h1, h2, h3, h4, h5, p, ul{
+    font-size: 1.6rem;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.container {
+    max-width: 880px;
+    width: 95%;
+	/* TB LR */
+	margin: 0 auto;
+  }
+  @media(max-width: 800px){
+    .container{
+        width: 95%;
+        margin: 0 auto;
+    }
+  }
+  
+  /* header */
+  
+  header {
+	  display:flex;
+	  justify-content: space-between
+
+  }
+  @media(max-width: 800px){
+    header{
+        flex-wrap: wrap;
+    }
+  }
+
+  header nav {
+	display: flex;
+	justify-content: space-between;
+	width: 68%;
+	vertical-align: top;
+	margin-top: 35px;
+  }
+  @media(max-width: 800px){
+      header nav{
+          order: 1;
+          width: 100%;
+          margin: 35px 2.5% 0 2.5%;
+      }
+  }
+  @media(max-width: 500px){
+    header nav{
+      flex-direction: column;
+      align-items: center;
+
+    }
+}
+
+  
+  header nav a {
+	
+	font-size: 1.4rem;
+	color: gray;
+	text-decoration: none;
+  }
+  @media(max-width: 500px){
```

Comment:
There are some spacing and indentation inconsistencies throughout. Again, try [prettier](https://prettier.io/); I swear by it!